.gitignore

```diff
@@ -2,4 +2,5 @@ screenshotTaker.jar
 adbScreenshotTest.png
 screenshotTakerTest.png
 build
+bin
```

Comment:
What is in the bin/ directory?

Reply:
During the tests we will have temp files in bin dir. If something goes wrong we can see several temp files inside git tree.